pkg/tls/certificatemanagement/certificatebundle.go

```diff
@@ -174,10 +174,9 @@ func (t *trustedBundle) ConfigMap(namespace string) *corev1.ConfigMap {
 
 	// Sort the certificates so that we get a consistent ordering.
 	// This reduces the number of changes we see in the configmap.
-	certs := []CertificateInterface{}
-	for _, cert := range t.certificates {
-		certs = append(certs, cert)
-	}
+	var certs []CertificateInterface
+	certs = append(certs, t.certificates...)
+
 	sort.Slice(certs, func(i, j int) bool {
```

Comment:
Is this sorting still needed ? If so, why can't we move this sorting after adding all certificates so that it is available for everybody ?

Reply:
Good point, I think that there should not be a need for it, as the order in which things are added to the bundle (which is now backed by a slice, rather than a map) is deterministic.

Reply:
We decided to keep the sorting regardless. (See last commit message).